<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wAddTerm</class>
 <widget class="QMainWindow" name="wAddTerm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wAddTerm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="ledID">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string notr="true">none</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblID">
      <property name="text">
       <string notr="true">ID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblArgument">
      <property name="text">
       <string notr="true">Term:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string notr="true">Save</string>
      </property>
      <property name="icon">
       <iconset resource="../../ressources.qrc">
        <normaloff>:/rec/data/icons/iconCheck.svg</normaloff>:/rec/data/icons/iconCheck.svg</iconset>
      </property>
      <property name="shortcut">
       <string notr="true">Return</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="pteArgument"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
